src/test/java/karate/article/Create.java

```diff
@@ -0,0 +1,10 @@
+package karate.article;
+
+import com.intuit.karate.junit5.Karate;
+
+public class Create {
+    @Karate.Test
+    Karate testSample() {
+        return Karate.run("classpath:karate/article/create.feature").relativeTo(getClass());
```

Comment:
remember you don't need to specify the feature.. why don't you delete this classes? What are you using that for?